_includes/android/geopoints.md

```diff
@@ -16,6 +16,42 @@ This point is then stored in the object as a regular field.
 placeObject.put("location", point);
 ```
 
+To retrieved `ParseGeoPoint` from an object.
```

Comment:
Still need to correct **retrieved** to **retrieve**. To improve the wording could be
> To **retrieve a** `ParseGeoPoint` from an object.